freshmaker/handlers/__init__.py

```diff
@@ -611,6 +611,9 @@ def build_image(build):
                 # We skip this image for now. It will be built once the ODCS
                 # compose is finished.
                 return
+            except Exception as ex:
```

Comment:
I'm thinking you should fail the build here with a state reason of `An unknown error occurred`. If you don't, then it'll say `Error while building container image in Koji.` which is misleading since the error may have happened outside of Koji.